Include/httpClient/httpClient.h

```diff
@@ -451,6 +451,20 @@ STDAPI HCHttpCallRequestSetTimeoutWindow(
     _In_ uint32_t timeoutWindowInSeconds
     ) noexcept;
 
+/// <summary>
+/// Enables or disables SSL server certificate validation for this specific HTTP call.
+/// Only implemented for Win32.
+/// The default is true
+///
+/// Optional, call prior to calling HCHttpCallPerformAsync.
+/// </summary>
+/// <param name="call">The handle of the HTTP call</param>
+/// <param name="traceCall">Validate certificates</param>
+/// <returns>Result code for this API operation. Possible values are S_OK, E_INVALIDARG, or E_FAIL.</returns>
+STDAPI HCHttpCallRequestSetSSLValidation(
+    _In_ HCCallHandle call,
+    _In_ bool sslValidation
+) noexcept;
```

Comment:
@ do we want to #ifdef this API to platforms that use winhttp? or just have it non-functional on other platforms?

Reply:
Hmm, yeah #ifdef for now around the public header until this is supported every would avoid complexity of someone using it and it not doing anything.  

the two platforms that use WinHTTP are Win32 & GDK.  

Cry-Luca, Can you wrap this public header and the src impl functions with:

#if HC_PLATFORM == HC_PLATFORM_WIN32 || HC_PLATFORM == HC_PLATFORM_GDK
#endif

Reply:
Sure will do that.